### ISSUE TITLE
Create a project from an open project

### ISSUE THREAD

**Issue by aplteam** [the-carlisle-group/Acre-Desktop#98]
Let's assume I have a project "C:\Foo"

I open it with `]acre.OpenProject "C:\Foo"` which creates a namespace `#.Foo`.

Now I want to create a project from `#.Foo` that lives elsewhere, so I do

```
      ]acre.CreateProject "C:\Temp\Foo" #.Foo
```

That works, and therefore I end up with two opened projects on `#.Foo`.

Maybe we need an `]acreCloseProject` after all?!

**Comment by PhilLast** [comment]
I'll knock this on the head but actually it won't cause a headache leaving it as it is.

If you change anything the fact that dyadic iota gives the first occurrence of whatever it's looking at means it'll be saved to the original project. If you reopen either of them then that will become the only open project.

Nevertheless I won't leave it as it is. In fact there is a decision to be made:

* either none of the project space of an open (tracked) project or a child or parent thereof should be source for a new project
* or creating a new project from the project space of an open project or a child or parent thereof should close (turn off tracking) the original.

**Comment by aplteam** [comment]
Well, the first option is not really an option because without an `]acre.Close` one would not be able to clone a project!

**Comment by PhilLast** [comment]
I agree that there's no point in prohibiting it if we can change it to work clearly and safely. But even at present we can do it without any risk or uncertainly. Here is more-or-less what you did:
```
      ]openproject .../old/ #.here
...
      ]projects
#.here .../old/
      ]createproject .../new/ #.here
...
      ]projects
#.here .../old/
#.here .../new/
```
Changes continue to go to `.../old/` so no actual danger so long as you know.
Here is what you could have done:
```
      ]openproject .../old/ #.here -t=off
...
      ]projects
No open projects
      ]createproject .../new/ #.here
...
      ]projects
#.here .../new/
```
And I should recommend that even after we change it. And by the way:
```
      ]openproject .../this/
...
      ]projects
#.this .../this/
      ]openprojects .../this/ -t=off
...
      ]projects
No open projects
```

**Comment by PhilLast** [comment]
Of course there are yet more alternatives. 

* `CreateProject` could just not open (track) a new project if its project space is a superset, set or subset of a tracked project.
* `CreateProject` could take notice of the `-track` modifier.

But given that cloning a project by using `CreateProject` leaves all files other than those in `APLSource` to be copied by means of the OS, using the OS to do the whole job would probably be more straightforward.
